src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
```

Comment:
` Support: IE 9 - 11` - https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-69379197

I'm OK with ditching the Chrome reference, since at this point it's just a historical curiosity.